xformers/ops/memory_efficient_attention.py

```diff
@@ -747,6 +773,7 @@ class AttentionOpDispatch:
     device: Union[torch.device, str]
     k: int
     has_dropout: bool
+    has_custom_scale: bool
```

Comment:
Some users call the constructor directly, so we should add a default value to not break backward compatibility
```suggestion
    has_custom_scale: bool = False
```

Reply:
Thanks for the good catch.